agent/agent-profiler/request-triggers.md

```diff
@@ -166,7 +166,7 @@ Currently, we support:
       "requestTriggerEndpoints": [
         {
           "name": "All",
-          "type": "latency",
+          "type": "LATENCY",
```

Comment:
we usually use lowercase for config enums, you can use `@JsonProperty("latency")` to make the json binding work